.travis.yml

```diff
@@ -32,3 +30,35 @@ jobs:
       script:
         - make
         - make sanity-test
+    - stage: test
+      name: "e2e test"
+      before_install:
+        - sudo apt-get -qq -y install conntrack
+      env:
+        - CHANGE_MINIKUBE_NONE_USER=true
+        - MINIKUBE_WANTUPDATENOTIFICATION=false
+        - MINIKUBE_WANTREPORTERRORPROMPT=false
+        - MINIKUBE_HOME=$HOME
+        - CHANGE_MINIKUBE_NONE_USER=true
+        - KUBECONFIG=$HOME/.kube/config
+      before_script:
+        - curl -Lo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.18.1/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
```

Comment:
is it possible to put these commands into one script file? so others could also set up minikube cluster in local env?

Reply:
Hi @andyzhangx I have made the changes. The script is present in `test/utils`. Thanks